pkg/commands/trait.go

```diff
@@ -76,8 +78,8 @@ func AddTraitCommands(parentCmd *cobra.Command, c types.Args, ioStreams cmdutil.
 		for _, v := range tmp.Parameters {
 			types.SetFlagBy(pluginCmd.Flags(), v)
 		}
-		pluginCmd.Flags().StringP(App, "a", "", "create or add into an existing application group")
-		pluginCmd.Flags().BoolP(Staging, "s", false, "only save changes locally without real update application")
+		pluginCmd.Flags().StringP(Service, "s", "", "specify one service belonging to the application")
```

Comment:
`--svc` is short enough, we don't need to occupy one more flag `s` widely.

Reply:
fixed.